smk-edit/static/components/presentation.js

```diff
@@ -74,6 +74,11 @@ export const toolTypePresentation = {
     //     icon: '',
     //     instance: false
     // },
+    'geomark': {
+        title: 'Geomark',
+        details: 'edit-tool-details-geomark',
+        default: {}
```

Comment:
I notice that some other tools have their icon specified in this 'default' object. Any idea why that is?

Reply:
The default icon is the spanner, so my understanding is that any time you want an icon that's not a spanner, you then specify it. I couldn't think of a reason not to go with the spanner.

Reply:
Cool, thanks for the explanation!